ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/UserManagementServiceIT.scala

```diff
@@ -31,80 +31,25 @@ import com.daml.ledger.api.v1.admin.user_management_service.{
 }
 import com.daml.ledger.api.v1.admin.{user_management_service => proto}
 
-import scala.concurrent.Future
+import scala.concurrent.{ExecutionContext, Future}
 import scala.util.{Failure, Success}
 
 final class UserManagementServiceIT extends UserManagementServiceITBase {
 
   private val adminPermission =
     Permission(Permission.Kind.ParticipantAdmin(Permission.ParticipantAdmin()))
-  private val actAsPermission1 =
-    Permission(Permission.Kind.CanActAs(Permission.CanActAs("acting-party-1")))
-  private val readAsPermission1 =
-    Permission(Permission.Kind.CanReadAs(Permission.CanReadAs("reading-party-1")))
-  private val userRightsBatch = List(
-    actAsPermission1,
-    Permission(Permission.Kind.CanActAs(Permission.CanActAs("acting-party-2"))),
-    readAsPermission1,
-    Permission(Permission.Kind.CanReadAs(Permission.CanReadAs("reading-party-2"))),
+  private def actAsPermission1(suffix: String) =
+    Permission(Permission.Kind.CanActAs(Permission.CanActAs(s"acting-party-1-$suffix")))
+  private def readAsPermission1(suffix: String) =
+    Permission(Permission.Kind.CanReadAs(Permission.CanReadAs(s"reading-party-1-$suffix")))
+  private def userRightsBatch(suffix: String) = List(
+    actAsPermission1(suffix),
+    Permission(Permission.Kind.CanActAs(Permission.CanActAs(s"acting-party-2-$suffix"))),
+    readAsPermission1(suffix),
+    Permission(Permission.Kind.CanReadAs(Permission.CanReadAs(s"reading-party-2-$suffix"))),
   )
   private val AdminUserId = "participant_admin"
 
-  test(
```

Comment:
We're limiting to 1000 and I would need to  allocate 1000 parties which I was told would be ~100ms in a canton case. What would you suggest?

Reply:
The reason that this test is being deleted is that 100ms is too long?

Reply:
100ms * 1000, i.e 100 seconds. I could maybe in test make the limit dynamically to the smaller number and still do the test, not sure how complicated yet it could be.

Reply:
How about we restrict the test to ledgers where the number of rights per user is 0<number<=100?